app/javascript/pages/components/Gallery.jsx

```diff
@@ -39,7 +39,7 @@ class Gallery extends React.Component {
           <section className="container calendar-spotlight tight">
             <div className="calendar-spotlight__image-wrapper">
```

Comment:
Right now, only clicking on the actual text leads to the download---maybe we should apply the link to the image wrapper so both the text and the entire image space can be clicked?

Reply:
Good call, I've updated it.